Mathlib/Data/Finset/Lattice.lean

```diff
@@ -1385,7 +1385,7 @@ theorem max_of_nonempty {s : Finset α} (h : s.Nonempty) : ∃ a : α, s.max = a
 theorem max_eq_bot {s : Finset α} : s.max = ⊥ ↔ s = ∅ :=
   ⟨fun h ↦ s.eq_empty_or_nonempty.elim id fun H ↦ by
       obtain ⟨a, ha⟩ := max_of_nonempty H
-      rw [h] at ha; cases ha; done, -- Porting note: error without `done`
```

Comment:
What is going on here lol, and below

Reply:
For this, I'll direct you [here](https://leanprover.zulipchat.com/#narrow/stream/270676-lean4/topic/error.20puzzle)!